src/vehicle.cpp

```diff
@@ -3580,10 +4035,11 @@ void vehicle::refresh_insides()
                                           parts[p].mount.y + ndy );
             bool cover = false; // if we aren't covered from sides, the roof at p won't save us
             for( auto &j : parts_n3ar ) {
-                if( part_flag( j, "ROOF" ) && !parts[ j ].is_broken() ) { // another roof -- cover
+                // another roof -- cover
+                if( part_flag( j, "ROOF" ) && !parts[ j ].is_available() ) {
```

Comment:
! before `is_available` should also be removed here

Reply:
Fixed.